sites/start.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>start</title>
+    <link href ="../styles/normalize.css" rel="stylesheet">
+    <link href ="../styles/styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <section id="hero">
+        <img src="../images/mommy.jpg" class = "bounce-in-top" alt="hero image">
+        <h1 class = "dropshadow">It's Me!</h1>
+        <p class="center-p">
```

Comment:
I'm a bit confused about this `start.html` and `index.html`. Which one is the homepage?